assets/src/js/admin/reports/components/ProductRecommendation/index.tsx

```diff
@@ -0,0 +1,81 @@
+import {ReactElement, useState} from 'react';
+import {__} from '@wordpress/i18n';
+import {createInterpolateElement} from '@wordpress/element';
+import {getWindowData} from '../../utils';
+import './style.scss';
+
+/**
+ * @unreleased
+ */
+export default function ProductRecommendation() {
+    const [showRecommendation, setShowRecommendation] = useState<boolean>(true);
+
+    const pluginUrl = getWindowData('pluginUrl');
+
+    const closeMessage = async (async) => {
+        await removeRecommendation();
+        setShowRecommendation(false);
+    };
+
+    return (
+        showRecommendation && (
+            <div className={'givewp-reports-recurring-recommendation'}>
+                <div className={'givewp-reports-recurring-recommendation-container'}>
+                    <div>
+                        <img src={`${pluginUrl}/assets/dist/images/list-table/light-bulb-icon.svg`} />
+
+                        <TranslatedMessage />
+                    </div>
+
+                    <a target="_blank" href={'https://docs.givewp.com/subscriptions'}>
+                        {__('Get more donations')}
+                        <img src={`${pluginUrl}/assets/dist/images/list-table/external-link-icon.svg`} />
+                    </a>
+                </div>
+
+                <button onClick={closeMessage}>
+                    <img src={`${pluginUrl}/assets/dist/images/close-icon.svg`} />
+                </button>
+            </div>
+        )
+    );
+}
+
+/**
+ * @unreleased
+ */
+function TranslatedMessage() {
+    const ProTip = <strong>Pro Tip: </strong>;
+
+    const Message: () => ReactElement<string> = () => {
+        return createInterpolateElement(
+            __('<ProTip/> Increase your fundraising revenue by over 30% with recurring giving campaigns.'),
+            {ProTip}
+        );
+    };
+
+    return <Message />;
```

Comment:
I don't think we need an interpolated element since `<strong>` has no attributes. Also, "Pro Tip" wasn't being made translatable.

```jsx
return (
    __('<strong>Pro Tip:</strong> Increase your fundraising revenue by over 30% with recurring giving campaigns.')
);
```

In fact, this can probably just be moved inline.

Reply:
It's been updated to include a message for fee recovery. Resolved: https://github.com/impress-org/givewp/pull/6796/commits/c8db2c4961a553f457d84edd893a9ce852a01859